### PR TITLE
Remove autoload.php

### PR DESCRIPTION
**Pull request by Art4**
The `src/autoload.php` was created in #96 to allow using the library without Composer. 

Since v1.7.0 we have dependencies to `psr/http-client` and `psr/http-factory` so the `src/autoload.php` stops working without manual installing this dependencies. The new recommended way [to use this library without Composer](https://github.com/kbsali/php-redmine-api#manual) is to use a service like [php-download.com](https://php-download.com/package/kbsali/redmine-api). Therefore, `src/autoload.,php` is no longer needed.

This PR removes the `src/autoload.php`.

### CODE REVIEW

**Review by kbsali — APPROVED** [review]

**Comment by Art4** [comment]
In case someone should ask for an installation possibility by hand (without Composer), please follow these instruction:
https://stackoverflow.com/a/40545971

It can be hard to follow the instruction, especially when there are many dependencies with sub-dependencies. (This work is usually done for you by Composer.)

So I have created a script `install-redmine-api.sh` (based on [this instruction](https://getcomposer.org/doc/faqs/how-to-install-composer-programmatically.md)) that downloads the library with all dependencies for you. It will create a directory called `vendor` and generates the `autoload.php` file inside this vendor directory. Installation of **PHP is required**.

### install-redmine-api.sh

```sh
#!/bin/sh

>&2 echo 'Download composer.phar file to local directory'

EXPECTED_CHECKSUM="$(php -r 'copy("https://composer.github.io/installer.sig", "php://stdout");')"
php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
ACTUAL_CHECKSUM="$(php -r "echo hash_file('sha384', 'composer-setup.php');")"

if [ "$EXPECTED_CHECKSUM" != "$ACTUAL_CHECKSUM" ]
then
    >&2 echo 'ERROR: Invalid installer checksum'
    rm composer-setup.php
    exit 1
fi

php composer-setup.php --quiet
RESULT=$?
rm composer-setup.php

if [ "$RESULT" != 0 ]
then
    >&2 echo 'ERROR: Could not install composer.phar'
    rm composer.phar
    exit $RESULT
fi

INSTALL_VERSION=""

if [[ $1 != "" ]]; then
    INSTALL_VERSION=":"$1;
fi

>&2 echo 'Download PHP library kbsali/php-redmine-api and create autoload.php'

php composer.phar require kbsali/redmine-api$INSTALL_VERSION --quiet
RESULT=$?

if [ "$RESULT" != 0 ]
then
    >&2 echo ''
    >&2 echo 'ERROR: Could not download kbsali/php-redmine-api'
    rm composer.json
    rm composer.phar
    exit $RESULT
fi

>&2 echo 'Remove composer.phar from local directory and other unused files'

rm composer.json
rm composer.lock
rm composer.phar

>&2 echo ''
>&2 echo 'Alright! You can now start using kbsali/php-redmine-api in your PHP script:'
>&2 echo 'require_once("vendor/autoload.php");'
```

### Usage

Download the latest version:

```sh
$ ./install-redmine-api.sh
```

or download a specific version:

```sh
$ ./install-redmine-api.sh v1.8.1
```

### Output

```sh
$ ./install-redmine-api.sh
Download composer.phar file to local directory
Download PHP library kbsali/php-redmine-api and create autoload.php
Remove composer.phar from local directory and other unused files

Alright! You can now start using kbsali/php-redmine-api in your PHP script:
require_once("vendor/autoload.php");
```
